director/director.go

```diff
@@ -397,8 +397,15 @@ func redirectToOrigin(ginCtx *gin.Context) {
 	reqPath := path.Clean("/" + ginCtx.Request.URL.Path)
 	reqPath = strings.TrimPrefix(reqPath, "/api/v1.0/director/origin")
 
+	disableStat := param.Director_DisableStat.GetBool()
```

Comment:
Can we change the config name to match the style we use elsewhere, i.e. `Director.EnableStat` and then flip the default to be `true`?

Reply:
I thought the idea/guideline is to name the config in a way that the default value is `false`? No?

Reply:
Hmm, I don't know that we've settled on a formal guideline, have we? Looking over `docs/parameters.yaml` shows that we're mostly using `.Enable*`-type parameters (some of which default to true, and some to false), although we do have one "Disable" param that doesn't come from a legacy StashCP configuration (`Logging.DisableProgressBars`). If you prefer the param as is, I'll approve and let you make the decision.

Reply:
I'll do `Enable*` then just to keep things consistent